ts/fast-sync/types.ts

```diff
@@ -5,11 +5,16 @@ export interface FastSyncSenderChannel {
 }
 export interface FastSyncReceiverChannel {
     streamObjectBatches : () => AsyncIterableIterator<FastSyncBatch>
+    receiveSyncInfo: () => Promise<FastSyncInfo>
 }
 export interface FastSyncInfo {
     objectCount : number
     collectionCount : number
 }
+export interface FastSyncProgress {
```

Comment:
Would be nice if this could extend `FastSyncInfo`. Would make it easer for the UI to calculate progress without needing to remember much itself.

Reply:
Yep! good point - put enough information here to be most useful to the calling code (UI). Have now made `FastSyncProgress extend FastSyncInfo` as suggested.